app/controllers/admin/communities_controller.rb

```diff
@@ -266,6 +285,8 @@ def update_settings
 
     maybe_update_payment_settings(@current_community.id, params[:community][:automatic_confirmation_after_days])
 
+    MarketplaceService::API::Marketplaces.update_marketplace_configurations(community_id: @current_community.id, main_search: params[:main_search]) if feature_enabled?(:location_search)
```

Comment:
The namespace (Marketplace) is repeated two times here. Could there be a simple and shorter name with less repetition, e.g. `MarketplaceService::API::Configuration.update( ... )`?